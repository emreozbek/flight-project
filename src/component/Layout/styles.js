export const styles = theme => ({
  background: {
    display: "block",
    content: "",
    width: "100%",
    height: theme.spacing.unit * 30,
    backgroundColor: theme.palette.secondary.main,
    position: "absolute",
    zIndex: -1
  },
  wrapper: {
    padding: `${theme.spacing.unit * 8}px ${theme.spacing.unit * 2}px 0 ${theme
      .spacing.unit * 2}px`,
    maxWidth: 720 + theme.spacing.unit * 4,
    marginLeft: "auto",
    marginRight: "auto"
  }
});
