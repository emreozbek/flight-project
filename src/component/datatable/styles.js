export const styles = theme => ({
  table: {
    minWidth: 720
  },
  tableWrapper: {
    overflowX: "auto"
  },
  searchBox: {
    paddingTop: theme.spacing.unit * 2,
    paddingLeft: theme.spacing.unit * 2,
    paddingRight: theme.spacing.unit * 2,
    paddingBottom: theme.spacing.unit * 2
  }
});
