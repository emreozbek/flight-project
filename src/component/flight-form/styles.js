export const styles = theme => ({
  padding: {
    padding: theme.spacing.unit * 2
  },
  marginLeft: {
    marginLeft: theme.spacing.unit * 2
  }
});
