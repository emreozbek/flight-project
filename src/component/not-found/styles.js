export const styles = theme => ({
  padding: {
    padding: theme.spacing.unit * 2
  }
});
