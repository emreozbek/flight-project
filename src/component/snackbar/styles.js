export const styles = theme => ({
  close: {
    padding: theme.spacing.unit / 2
  },
  error: {
    backgroundColor: theme.palette.error.main
  }
});
