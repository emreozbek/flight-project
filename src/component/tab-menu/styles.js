export const styles = theme => ({
  content: {
    position: "relative",
    minHeight: 256
  }
});
