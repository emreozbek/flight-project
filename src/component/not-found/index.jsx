import React, { Fragment } from "react";
import { Link } from "react-router-dom";
import PropTypes from "prop-types";
import { withStyles, Typography } from "@material-ui/core";
import { styles } from "./styles";

function NotFound(props) {
  const { classes } = props;
  return (
    <Fragment>
      <Typography variant="h4" className={classes.padding} align="center">
        404 Not Found
      </Typography>
      <Typography
        variant="subtitle1"
        className={classes.padding}
        align="center"
      >
        You should turn back to <Link to="/">Home</Link>
      </Typography>
    </Fragment>
  );
}

NotFound.propTypes = {
  classes: PropTypes.object.isRequired
};
export default withStyles(styles)(NotFound);
