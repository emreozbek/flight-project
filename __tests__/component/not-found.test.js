import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import { withStyles } from "@material-ui/core";
import NotFound from "../../src/component/not-found";
import { styles } from "../../src/component/not-found/styles";

const NotFoundStyled = withStyles(styles)(NotFound);
describe("Not Found", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <NotFoundStyled />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
