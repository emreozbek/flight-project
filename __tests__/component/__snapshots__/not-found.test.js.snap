// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found renders correctly 1`] = `
Array [
  <h4
    className="MuiTypography-root-3 MuiTypography-h4-18 MuiTypography-alignCenter-26 NotFound-padding-2 WithStyles-NotFound--padding-1"
  >
    404 Not Found
  </h4>,
  <h6
    className="MuiTypography-root-3 MuiTypography-subtitle1-21 MuiTypography-alignCenter-26 NotFound-padding-2 WithStyles-NotFound--padding-1"
  >
    You should turn back to 
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </h6>,
]
`;
